planning_node:
  ros__parameters:
    use_sim_time: true
    controller_name: "fanuc_m20ia_35m_PID_controller"
    controller_type: "JointTrajectoryController"
    max_velocity_scaling_factor: 1.0